### ISSUE TITLE
Trap button cloning itself

### ISSUE THREAD

**Issue by derpedcatto** [Zet0rz/nzombies#487]
When i restart my game and load up my config trap button is cloning itself.
It only happens if i close gmod and start it again, going on my config.
![cs_italy0009](https://cloud.githubusercontent.com/assets/29012318/26527474/68857d34-439d-11e7-8490-a48585ca7be9.png)
![cs_italy0010](https://cloud.githubusercontent.com/assets/29012318/26527475/68861a14-439d-11e7-9682-e72e26c6137c.png)

**Comment by Tristanbijl** [comment]
Hi. Sorry to necropost. But did you ever find a fix for this?

**Comment by derpedcatto** [comment]
Nope, but i haven't encountered this for a while for some reason. All i know what this happens not on every config and happens after every config load (i hope you understand lol).

**Comment by DjNathann** [comment]
The gamemode is in beta so expect bugs like this,
